### PR TITLE
Update logo and top navigation

### PR DESCRIPTION
**Pull request by techwriter-dev**
- Removed top navigation options
- Added new word-only logo and renamed the file to be more descriptive
- Replaced favicon with a light-mode emnify logo (blue on white background)

### CODE REVIEW

**Comment by techwriter-dev** [comment]
@carolstran Is there anything else that needs to be done? Thanks!

File: docusaurus.config.js [hunk 1]
```diff
@@ -59,25 +59,11 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       navbar: {
-        title: "Home",
+        title: "Developer",
```
Comment (carolstran) [inline docusaurus.config.js:62]: IMO this title should probably reflect the domain we choose, but either way this should be plural:

```suggestion
        title: "Developers",
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -59,25 +59,11 @@ const config = {
...
         logo: {
           alt: "EMnify Logo",
-          src: "img/Logo-web_logo-white-onblue.svg",
+          src: "img/logo-word-blue-295x80.png",
```
Comment (carolstran) [inline docusaurus.config.js:65]: We should also add the logo for dark mode as part of this PR. 

Looking at the [Docusaurus docs](https://docusaurus.io/docs/api/themes/configuration#navbar-logo), we can use the `srcDark` field for this.

File: docusaurus.config.js [hunk 3]
```diff
@@ -59,25 +59,11 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       navbar: {
-        title: "Home",
+        title: "Developer",
         logo: {
           alt: "EMnify Logo",
```
Comment (carolstran) [inline docusaurus.config.js:64]: It'd be great to add more descriptive alt text here:

```suggestion
          alt: "emnify Documentation – Home",
```

(could also be `Developer Documentation` or some other variation, up to you)

Reference: [Deque](https://www.deque.com/blog/great-alt-text-introduction/#:~:text=At%20a%20minimum%2C%20the%20alt,%E2%80%9CUser%20Testing%20%E2%80%93%20Home%E2%80%9D.)

**Review by carolstran — CHANGES_REQUESTED** [review]
Some details need updating before merging, but the styling looks great thank you ✨

File: docusaurus.config.js (hunk 2 above)
Comment (techwriter-dev) [reply docusaurus.config.js]: Resolved in [commit 73a0a869](https://github.com/EMnify/product-docs/pull/13/commits/73a0a86995e8cb1a9c97a004b4df549f06ddc17e)

**Review by carolstran — APPROVED** [review]
Looks great, thank you 🎉

**Pull request merged**